logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/usecase/UpdateVideoStateUseCase.kt

```diff
@@ -16,14 +16,14 @@ class UpdateVideoStateUseCase(
         conversationId: ConversationId,
         videoState: VideoState
     ) {
-        callRepository.updateIsCameraOnById(conversationId.toString(), videoState == VideoState.STARTED)
+        if (videoState != VideoState.PAUSED)
+            callRepository.updateIsCameraOnById(conversationId.toString(), videoState == VideoState.STARTED)
 
-        //updateVideoState should be called only when the call is answered/established
+        // updateVideoState should be called only when the call is answered/established
         callRepository.callsFlow().first().find { call ->
```

Comment:
You can remove the verification of the call status to be `ESTABLISHED` if we use the `ongoingCallsFlow` directly instead of going through all the calls we have.

```suggestion
        callRepository.ongoingCallsFlow().first().find { call ->
```